### PR TITLE
Update to new lit-element API

### PR DESCRIPTION
**Pull request by frankiefu**
update to `lit-element#dev`

### CODE REVIEW

File: packages/button/mwc-button.js [hunk 1]
```diff
@@ -14,7 +14,8 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-import {LitElement, html, classString as c$} from '@polymer/lit-element/lit-element.js';
+import {LitElement, html} from '@polymer/lit-element/lit-element.js';
+import {classString as c$} from '@polymer/lit-element/lib/render-helpers.js';
```
Comment (sorvell) [inline packages/button/mwc-button.js]: Being removed from LitElement. Will be included in lit-html as `classMap`.

Comment (sorvell) [reply packages/button/mwc-button.js]: Let's roll our own and change when this gets merged to lit-html.

File: packages/card/mwc-card.js [hunk 2]
```diff
@@ -29,14 +29,14 @@ export class Card extends LitElement {
...
     return style;
   }
 
-  _render({stroke}) {
+  render() {
     return html`
-      ${this._renderStyle()}
-      <div class$="mdc-card ${stroke ? 'mdc-card--stroked' : ''}">
+      ${this.renderStyle()}
+      <div class="mdc-card ${this.stroke ? 'mdc-card--stroked' : ''}">
```
Comment (sorvell) [inline packages/card/mwc-card.js]: Use `classMap` for this.

File: packages/chips/mwc-chip.js [hunk 3]
```diff
@@ -48,19 +48,22 @@ export class Chip extends ComponentElement {
...
+      html`<i class="material-icons mdc-chip__icon mdc-chip__icon--leading">${leadingIcon}</i>` : '';
+    const trailingIconPart = trailingIcon ?
+      html`<i class="material-icons mdc-chip__icon mdc-chip__icon--trailing">${trailingIcon}</i>` : '';
     // TODO(sorvell) #css: added display
     return html`
-      ${this._renderStyle()}
-      <div class$="mdc-chip ${this.active ? 'mdc-chip--activated' : ''}"
-        on-MDCChip:interaction="${this._boundInteractionHandler}">
+      ${this.renderStyle()}
+      <div class="mdc-chip ${this.active ? 'mdc-chip--activated' : ''}"
```
Comment (sorvell) [inline packages/chips/mwc-chip.js]: use `classMap`

File: packages/dialog/mwc-dialog.js [hunk 4]
```diff
@@ -90,7 +91,7 @@ export class Dialog extends ComponentElement {
             <h2 id="my-mdc-dialog-label" class="mdc-dialog__header__title">${headerLabel}</h2>
             <slot name="header"></slot>
           </header>
-          <section id="my-mdc-dialog-description" class$="mdc-dialog__body ${scrollable ? 'mdc-dialog__body--scrollable' : ''}">
+          <section id="my-mdc-dialog-description" class="mdc-dialog__body ${scrollable ? 'mdc-dialog__body--scrollable' : ''}">
```
Comment (sorvell) [inline packages/dialog/mwc-dialog.js]: use `classMap`

File: packages/formfield/mwc-formfield.js [hunk 5]
```diff
@@ -44,34 +43,32 @@ export class Formfield extends ComponentElement {
...
     return style;
   }
 
-  _render({label, alignEnd, _labelClickHandler}) {
-    return html`${this._renderStyle()}
-      <div class$="mdc-form-field ${alignEnd ? 'mdc-form-field--align-end' : ''}">
+  render() {
+    const {label, alignEnd} = this;
+    return html`${this.renderStyle()}
+      <div class="mdc-form-field ${alignEnd ? 'mdc-form-field--align-end' : ''}">
```
Comment (sorvell) [inline packages/formfield/mwc-formfield.js]: use `classMap`

File: packages/select/mwc-select.js [hunk 6]
```diff
@@ -78,7 +79,7 @@ export class Select extends LitElement {
         }
       </style>
       ${menuStyle}${style}
-      <div class$="mdc-select ${box ? 'mdc-select--box' : ''}" role="listbox" aria-disabled$="${disabled}">
+      <div class="mdc-select ${box ? 'mdc-select--box' : ''}" role="listbox" aria-disabled="${disabled}">
```
Comment (sorvell) [inline packages/select/mwc-select.js]: use `classMap`

File: packages/slider/mwc-slider.js [hunk 7]
```diff
@@ -15,7 +15,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 import {FormableComponentElement, MDCWebComponentMixin, html} from '@material/mwc-base/formable-component-element.js';
-import {classString as c$} from '@polymer/lit-element/lit-element.js';
+import {classString as c$} from '@polymer/lit-element/lib/render-helpers.js';
```
Comment (sorvell) [inline packages/slider/mwc-slider.js]: use `classMap`

File: packages/textfield/mwc-textfield.js [hunk 8]
```diff
@@ -15,7 +15,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 import {ComponentElement, html} from '@material/mwc-base/component-element.js';
-import {classString as c$} from '@polymer/lit-element/lit-element.js';
+import {classString as c$} from '@polymer/lit-element/lib/render-helpers.js';
```
Comment (sorvell) [inline packages/textfield/mwc-textfield.js]: use `classMap`

File: packages/icon-toggle/mwc-icon-toggle.js [hunk 9]
```diff
@@ -57,36 +57,36 @@ export class IconToggle extends ComponentElement {
...
   // TODO(sorvell) #css: added display
-  _render({on, disabled, icon, offIcon, label, offLabel}) {
-    offIcon = offIcon || icon;
+  render() {
+    const {disabled, icon, offIcon, label, offLabel} = this;
     return html`
-      ${this._renderStyle()}
-      <span class$="mdc-icon-toggle material-icons ${disabled ? 'mdc-icon-toggle--disabled' : ''}"
+      ${this.renderStyle()}
+      <span class="mdc-icon-toggle material-icons ${disabled ? 'mdc-icon-toggle--disabled' : ''}"
```
Comment (sorvell) [inline packages/icon-toggle/mwc-icon-toggle.js]: let's use `classMap` here also

Comment (frankiefu) [reply packages/icon-toggle/mwc-icon-toggle.js]: Good catch! Done!

**Pull request merged**